### ISSUE TITLE
sorting a list of lists gives empty list!

### ISSUE THREAD

**Issue by BruceEdmonds** [NetLogo/NetLogo#439]
e.g. (in 5.0.4)
show sort [[1 2 3] [4 3 2]]
[]

(PS.  I suspect whatever causes this causes rare bugs with table extension when using lists as keys - its either that or I am going mad!)

**Comment by ERSUCC** [comment]
I agree that lists would intuitively be sorted recursively, and I see no possible error case. No matter what, every data type can somehow be converted to a string, and therefore lexicographically sorted. Maybe it won't give you the desired result, but it won't error, at least.
